modules/using-cli.adoc

```diff
@@ -122,6 +122,81 @@ You can remove them by running the `delete-sensor.sh` script from the Sensor ins
 [id="check-policy-compliance_{context}"]
 == Checking policy compliance
 
+You can use the `roxctl` CLI to check deployment YAML files and images for policy compliance.
+
+[discrete]
+=== Configuring output format
+When you check policy compliance by using the `deployment check`, `image check`, or `image scan` commands, you can specify the output format by using the `-o` option. This option determins how the output of a command is displayed in the terminal.
+
+You can change the output format by adding the `-o` option to the command and specifying the format as either `json`, `table`, `csv`, or `junit`.
+
+For examle, the following command checks a deployment and then displays the result in `csv` format:
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" \
+  deployment check -file =<yaml_filename> \
+  -o csv
+----
+
+[NOTE]
+====
+When you do not specify the `-o` option for the output format:
+
+* The default output format for the `deployment check` and the `image check` commands is `table`.
+* The default output format for the `image scan` command is `json`. This is the old JSON format output for compatibility with older versions of the CLI. To get the output in the new JSON format, specify the option with format, as `-o json`.
+====
+
+Depending on the format you select, there are different options available to configure the output. The following tabele lists the options for each format.
+
+[%header,cols="1,1,2"]
+|===
+|Format
+|Options
+|Description
+
+|`json`
+|`--compact-output`
+|Displays the JSON output in a compact format.
+
+.4+|`table`
+|`--headers`
+|Use this option to specify custom headers.
+
+|`--merge-output`
+|Use this options to merge table cells that have the same value.
+
+|`--no-header`
+|Use this option to omit the header row from the output.
+
+|`--row-jsonpath-expressions`
+a|Use this option to specify link:https://github.com/tidwall/gjson[GJSON paths] to select specific items from the output. For example, to get the *Policy name* and *Severity* for a deplyment check, use the following command:
```

Comment:
```suggestion
a|Use this option to specify link:https://github.com/tidwall/gjson[GJSON paths] to select specific items from the output. For example, to get the *Policy name* and *Severity* for a deployment check, use the following command:
```
OCP has a blanket ban on github links without clearance by support. (They're concerned about the appearance of extending supportability to uncontrolled sources.) Consider if additional approval would be beneficial here.